src/api/ledger/parse/suspended-payment-finish.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+'use strict';
+const assert = require('assert');
+const sjclcodec = require('sjcl-codec');
+const utils = require('./utils');
+
+function convertHexToString(hexString) {
+  const bits = sjclcodec.hex.toBits(hexString);
+  return sjclcodec.utf8String.fromBits(bits);
+}
+
+function parsePayment(tx: Object): Object {
+  assert(tx.TransactionType === 'SuspendedPaymentFinish');
+
+  return utils.removeUndefined({
+    memos: utils.parsePaymentMemos(tx),
+    owner: tx.Owner,
+    offerSequence: tx.OfferSequence,
```

Comment:
> I think that using different names for the same field would be more confusing, not less -- though I agree that the name OfferSequence is misleading.

This API is supposed to provide a level of abstraction over rippled, so there's no reason to preserve rippled naming; all names have been chosen to be the most clear name we could think of.